src/fontem.c

```diff
@@ -349,11 +358,60 @@ static void store_bitmap(FILE *c, FT_Bitmap *bitmap, char *bname, wchar_t ch, in
 			}
 			free(compressed_data);
 		} else {
-			for (unsigned int y = 0; y < (unsigned int)bitmap->rows; y++) {
-				fprintf(c, "\t");
-				for (unsigned int x = 0; x < (unsigned int)bitmap->width; x++)
-					fprintf(c, "0x%02x, ", (unsigned char)bitmap->buffer[y * bitmap->width + x]);
-				fprintf(c, "\n");
+			if(rotate) {
+				if(mono) {
+					unsigned int byte_rows = bitmap->rows / 8;
+					if (bitmap->rows & 7 ) byte_rows++;
+
+					unsigned char *out = malloc(byte_rows * bitmap->width);
+					memset(out,0,byte_rows * bitmap->width);
+
+					unsigned int x,y;
+
+					printf("\n");
```

Comment:
This, and the other `printf` statements below, should be `fprintf(c, ...)` in the case that the file handle `c` points to something other than `stdout`.